src/modules/pv/pv_core.c

```diff
@@ -3238,11 +3238,11 @@ int pv_get_msg_attrs(sip_msg_t *msg, pv_param_t *param, pv_value_t *res)
 
 int pv_parse_env_name(pv_spec_p sp, str *in)
 {
+	char *csname;
+
 	if(in->s==NULL || in->len<=0)
```

Comment:
I'll document it on the wiki when/if it makes its way to master eventually (after the feature freeze?)